tests/modules/images/webp-uploads/webp-uploads-test.php

```diff
@@ -17,7 +20,9 @@ function test_webp_uploads_applies_filter_based_on_system_support() {
 		} else {
 			$expect = array( 'image/jpeg' => 'image/webp' );
 		}
-
+		// The image_editor_output_format filter is only applied if image_editor_output_format
+		// has fewer than two types.
+		add_filter( 'image_editor_output_formats', array( $this, 'return_webp_array' ) );
```

Comment:
Why did you add this here? This existing test shouldn't be affected by this change, or is it?

Reply:
oh, you are right, I thought this was breaking the tests, removing.